### PR TITLE
feat: Add participant-item for use in lists

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: app/script/components/list/participantItem.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+window.z = window.z || {};
+window.z.components = z.components || {};
+
+z.components.ParticipantItem = class ParticipantItem {
+  constructor(params) {
+    this.participant = ko.unwrap(params.participant);
+    this.isService = this.participant instanceof z.integration.ServiceEntity || this.participant.isBot;
+    this.isUser = this.participant instanceof z.entity.User && !this.participant.isBot;
+    this.selfUser = window.wire.app.repository.user.self;
+    this.contentInfo = '';
```
Comment (gregor) [inline app/script/components/list/participantItem.js]: `this.contentInfo = this.isService ? this.participant.summary : this.participant.username()` and set the default value of username in L134 of the User entity to '';

File: app/style/panel/participants.less [hunk 2]
```diff
@@ -207,11 +208,6 @@
     margin: 106px 18px;
     white-space: pre;
   }
-
-  user-list,
-  service-list {
-    margin-top: 16px;
```
Comment (gregor) [inline app/style/panel/participants.less:213]: Where did you move this to?

File: app/style/components/list/participant-item.less [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      .participant-item-content-info {
+        margin-top: 2px;
+        display: flex;
+        font-size: 12px;
+        line-height: 16px;
+        opacity: 0.56;
+      }
+    }
+  }
+}
```
Comment (gregor) [inline app/style/components/list/participant-item.less:86]: Let's make this more readable with some empty lines  after closing brackets if not followed by another closing bracket.

File: app/style/panel/participants.less (hunk 2 above)
Comment (AndyLnd) [reply app/style/panel/participants.less:213]: Nowhere. The top margin was there to compensate for the lack of spacing on top, as the content was aligned to the top of the containing div.

**Pull request merged**